### PR TITLE
Add requiredForLanguages metadata to manifest

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

**Review by ilia-kebets-sonarsource — APPROVED** [review]
LGTM

File: sonar-flex-plugin/pom.xml [hunk 1]
```diff
@@ -134,10 +134,12 @@
       <plugin>
         <groupId>org.sonarsource.sonar-packaging-maven-plugin</groupId>
         <artifactId>sonar-packaging-maven-plugin</artifactId>
+        <version>1.22.0.705</version>
```
Comment (ilia-kebets-sonarsource) [inline sonar-flex-plugin/pom.xml:137]: why do we need to specify the version here?

Comment (saberduck) [reply sonar-flex-plugin/pom.xml:137]: to use the latest release, eventually we should update parent pom to include this automatically

**Review by ilia-kebets-sonarsource — APPROVED** [review]
approved

**Pull request merged**